src/sjef/util/Job.cpp

```diff
@@ -51,8 +51,7 @@ std::tuple<bool, std::string, std::string> sjef::util::Job::push_rundir(int verb
   auto start_time = std::chrono::steady_clock::now();
   (*m_backend_command_server)("mkdir -p '" + m_remote_cache_directory + "'");
   const Shell& shell = Shell();
-  auto rsync_out = shell(command, verbosity);
-  shell.wait();
+  auto rsync_out = shell(command, true, ".", verbosity);
```

Comment:
Yes, this is actually a fix for a different bug, namely that this call to `shell()` is to an old interface where `verbosity` was the second parameter, and so incorrectly when `verbosity` is zero, the call specifies asynchronous.